module/VuFind/src/VuFind/Controller/CartController.php

```diff
@@ -246,6 +257,12 @@ public function emailAction()
             return $this->redirectToSource('error', 'bulk_noitems_advice');
         }
 
+        // Check if id limit is exceeded
+        $actionLimit = $this->config?->BulkActions?->limits?->email ?? 0;
```

Comment:
Maybe a `checkBulkActionLimit($action)` support method would make this code less repetitive.

I'm also a little concerned about defaulting this to 0, since users who fail to update your config.ini when upgrading will suddenly lose all bulk options. Maybe we should add a separate config setting for "default bulk limit" and default the default to 100. What do you think?

Reply:
> Maybe a checkBulkActionLimit($action) support method would make this code less repetitive.
Did this and added it to a new BulkActionTrait.

>I'm also a little concerned about defaulting this to 0, since users who fail to update your config.ini when upgrading will suddenly lose all bulk options. Maybe we should add a separate config setting for "default bulk limit" and default the default to 100.
Good Idea. Did this.